src/Arcus.POC.WebApi.Logging.Core/Correlation/HttpCorrelation.cs

```diff
@@ -109,20 +173,26 @@ public bool TryHttpCorrelate(out string errorMessage)
             string transactionId = DetermineTransactionId(transactionIds);
             string operationId, operationParentId = null;
 
-            if (_options.UpstreamService.ExtractFromRequest && TryGetUpstreamOperationId(httpContext, out operationParentId))
-            {
-                operationId = ExtractOperationIdFromOperationParentId(operationParentId);
-                if (operationId is null)
-                {
-                    _logger.LogError("No operation parent ID with a correct format could be found in the request header '{HeaderName}' which means no operation ID could be extracted", _options.UpstreamService.OperationParentIdHeaderName);
-                    errorMessage = $"No correlation operation ID could be extracted from upstream service's '{_options.UpstreamService.OperationParentIdHeaderName}' request header";
-                    return false;
-                }
-            }
-            else
-            {
-                operationId = DetermineOperationIdWithoutUpstreamService(httpContext);
-            }
+            // TODO: generate a new operation ID and use received operation ID as parent
+            // old:
+            //if (_options.UpstreamService.ExtractFromRequest && TryGetUpstreamOperationId(httpContext, out operationParentId))
+            //{
+            //    operationId = ExtractOperationIdFromOperationParentId(operationParentId);
+            //    if (operationId is null)
+            //    {
+            //        _logger.LogError("No operation parent ID with a correct format could be found in the request header '{HeaderName}' which means no operation ID could be extracted", _options.UpstreamService.OperationParentIdHeaderName);
+            //        errorMessage = $"No correlation operation ID could be extracted from upstream service's '{_options.UpstreamService.OperationParentIdHeaderName}' request header";
+            //        return false;
+            //    }
+            //}
+            //else
+            //{
+            //    operationId = DetermineOperationIdWithoutUpstreamService(httpContext);
+            //}
+            // new:
+            operationParentId = httpContext.Request.Headers[_options.UpstreamService.OperationParentIdHeaderName];
+            operationId = DetermineOperationIdWithoutUpstreamService(httpContext);
+            _logger.LogTrace("Correlation: {OperationId}, {TransactionId}, {OperationParentId}", operationId, transactionId, operationParentId);
```

Comment:
Currently not using this `|abc.def` format and just place the parent ID in the header as-is.

Reply:
I currently have no opinion if we should move away from that format or not.

Reply:
We could change it a bit that we still support the HTTP correlation standard in the communication but not in Application Insights.
I'm even wondering if that was ever even correct as this is HTTP-specific.